#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_cftoken.l-bison.dpatch by  <jeremie.corbier@resel.enst-bretagne.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adapts cftoken.l to bison instead of yacc output

@DPATCH@

--- orig/cftoken.l	2006-02-09 16:09:32.000000000 +0100
+++ dest/cftoken.l	2006-02-09 16:09:50.000000000 +0100
@@ -52,7 +52,7 @@
 #if defined(__NetBSD__) && __NetBSD_Version__ >= 106000000	/* XXX */
 #include "cfparse.h"
 #else
-#include "y.tab.h"
+#include "cfparse.tab.h"
 #endif
 
 #define YY_NO_UNPUT 1
